bentoml/service/__init__.py

```diff
@@ -86,6 +101,10 @@ def api_decorator(
     :param output: OutputAdapter instance of the inference API
     :param api_name: API name, default to the user-defined callback function's function
         name
+    :param route: Specify HTTP URL route of this inference API. By default,
```

Comment:
one minor issue here is that we want to enforce that all inference APIs in a BentoService must have distinct route. previously it was not an issue because Python enforces users to choose a different function name implicitly, but now we may want to do a validation somewhere

Reply:
maybe something we could do in `_config_inference_apis` below?